PurchasesTests/SubscriberAttributes/AttributionDataMigratorTests.swift

```diff
@@ -528,50 +569,55 @@ private enum KeyPresence {
-            expect((converted[subscriberAttribute] as! String)) == (original[attributionKey] as! String)
+            expect((converted[subscriberAttribute] as? String)) == (original[attributionKey] as? String)
         }
     }
 
     func checkCommonAttributes(in converted: [String: Any],
                                idfa: KeyPresence = .defaultValue,
                                idfv: KeyPresence = .defaultValue,
-                               ip: KeyPresence = .defaultValue) {
+                               ipKey: KeyPresence = .defaultValue) {
+        let idfaValue = converted[ReservedSubscriberAttribute.idfa.key]
         switch idfa {
         case .defaultValue:
-            expect((converted[ReservedSubscriberAttribute.idfa.key] as! String)) == AttributionDataMigratorTests.defaultIdfa
+            expect(idfaValue as? String) == AttributionDataMigratorTests.defaultIdfa
         case .nsNull:
-            expect(converted[ReservedSubscriberAttribute.idfa.key]).to(beAKindOf(NSNull.self))
+            expect(idfaValue).to(beAKindOf(NSNull.self))
         case .notPresent:
-            expect(converted[ReservedSubscriberAttribute.idfa.key]).to(beNil())
+            expect(idfaValue).to(beNil())
         }
+
+        let idfvValue = converted[ReservedSubscriberAttribute.idfv.key]
         switch idfv {
         case .defaultValue:
-            expect((converted[ReservedSubscriberAttribute.idfv.key] as! String)) == AttributionDataMigratorTests.defaultIdfv
+            expect(idfvValue as? String) == AttributionDataMigratorTests.defaultIdfv
         case .nsNull:
-            expect(converted[ReservedSubscriberAttribute.idfv.key]).to(beAKindOf(NSNull.self))
+            expect(idfvValue).to(beAKindOf(NSNull.self))
         case .notPresent:
-            expect(converted[ReservedSubscriberAttribute.idfv.key]).to(beNil())
+            expect(idfvValue).to(beNil())
         }
-        switch ip {
+
+        let ipValue = converted[ReservedSubscriberAttribute.ip.key]
+        switch ipKey {
         case .defaultValue:
-            expect((converted[ReservedSubscriberAttribute.ip.key] as! String)) == AttributionDataMigratorTests.defaultIp
+            expect(ipValue as? String) == AttributionDataMigratorTests.defaultIp
         case .nsNull:
-            expect(converted[ReservedSubscriberAttribute.ip.key]).to(beAKindOf(NSNull.self))
+            expect(ipValue).to(beAKindOf(NSNull.self))
         case .notPresent:
-            expect(converted[ReservedSubscriberAttribute.ip.key]).to(beNil())
+            expect(ipValue).to(beNil())
         }
     }
 
     func adjustData(withIdfa idfa: KeyPresence = .defaultValue,
                     adjustId: KeyPresence = .defaultValue,
                     networkID: KeyPresence = .notPresent,
                     idfv: KeyPresence = .defaultValue,
-                    ip: KeyPresence = .defaultValue,
+                    ipKey: KeyPresence = .defaultValue,
```

Comment:
I would have been fine with disabling the check for this particular identifier, but I'm not opposed to `ipKey`

Reply:
Yeah, I could disable it for this file, ok?